src/core/textrenderer/qgstextformat.h

```diff
@@ -360,6 +360,19 @@ class CORE_EXPORT QgsTextFormat
      */
     double opacity() const;
 
+    /**
+     * Multiply opacity by \a opacityFactor.
+     *
+     * This method multiplies the opacity of all the labeling elements (text, shadow, buffer etc.)
+     * by \a opacityFactor effectively changing the opacity of the whole labeling.
+     *
+     * It is currently used by the WMS server to apply the OPACITIES parameters to the layer rendering
+     * of a GetMap request.
```

Comment:
Let's drop this -- I don't like having things like this in the documentation which are highly likely to become outdated information very quickly.